src/components/radio-group/daikin-radio-group.ts

```diff
@@ -88,13 +90,31 @@ export class DaikinRadioGroup extends LitElement {
   @property({ type: Boolean, reflect: true })
   required = false;
 
+  /**
+   * Whether the radio group is disabled.
+   * Controlled by `daikin-input-group` when used within `daikin-input-group`.
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
   /**
    * The label text used as the value of aria-label.
    * Set automatically by `reflectInputGroup` method.
    */
   @state()
   private _label: string | null = null;
 
+  @queryAssignedElements({
+    selector: "daikin-radio",
+  })
+  private readonly _controls!: readonly ControlElement[];
+
+  private _reflectSlotProperties(): void {
+    for (const control of this._controls) {
+      control.reflectInputGroup(this);
+    }
```

Comment:
Just update a prop instead of having a separate method.

```suggestion
    for (const radio of this _radios) {
      radio.disabledByParent = this.disabled;
    }
```